src/pages/Loans/LoansOverview/components/LoansTables/LoansTables.tsx

```diff
@@ -19,10 +19,23 @@ type LoansTablesProps = {
 const LoansTables = ({ lendPositions, borrowPositions, assets }: LoansTablesProps): JSX.Element => {
   const disabledAssets = useMemo(() => getDisabledAssetsTicker(assets), [assets]);
 
+  const hasBorrowPositions = !!borrowPositions.length;
+  const hasLendPositions = !!lendPositions.length;
```

Comment:
Since in both `LendTables` and `BorrowTables` we can simplyify the rendering condition to `hasBorrowPositions || hasLendPositions`, we can just generalize this to 
`const hasPositions = lendPositions.length > 0 && borrowPositions.length >0` and pass the same prop to both `LendTables` and `BorrowTables` components.